package.json

```diff
@@ -94,7 +94,9 @@
     "webpack-dev-server": "^1.14.1"
   },
   "dependencies": {
+    "babel-runtime": "^6.20.0",
     "blueimp-load-image": "^2.6.1",
+    "canvasfilters": "git://github.com/kig/canvasfilters.git",
```

Comment:
Can't you take the required .js file from here, and put it in the same place as the `measure_blur.js` script (again, a `lib/` folder would make sense to me)

Then you can `dependencies: { "canvasfilters": "file: ./lib/canvasfilters" }`

Just a bit more consistent about how we're pulling in non-npm-managed 3rd-party scripts